frontend/src/components/profile/EditProfile.vue

```diff
@@ -153,30 +144,24 @@ export default {
     methods: {
         ...mapActions({
             updateProfile: 'profile/updateProfile',
-            deleteUserAvatar: 'profile/deleteUserAvatar'
+            deleteUserAvatar: 'profile/deleteUserAvatar',
+            fetchProfileUser: 'profile/fetchProfileUser'
         }),
+        setProfileUser() {
+            this.fetchProfileUser(this.authUser.id)
```

Comment:
could we try a bit another way [https://gist.github.com/nemoipaha/31ae47edba89cdbdd64278169c92e37d](https://gist.github.com/nemoipaha/31ae47edba89cdbdd64278169c92e37d)  ?)